modulus/datapipes/__init__.py

```diff
@@ -13,3 +13,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from . import healpix
```

Comment:
Hi @yairchn, to follow convention for the rest of the datapipes, I am inclined to suggest removing this import statement. The reason is that each of the submodule datapipes may require various non-required packages and putting them in this level will require all `modulus.datapipe` users to have those dependencies. In this case, the use of `omegaconf` is an optional dependency in modulus but required for `healpix` datapipes.